test_suite/generic/test_generic.py

```diff
@@ -585,11 +585,12 @@ def test_cloud_cfg(self, host):
         """
         Verify file /etc/cloud/cloud.cfg is not changed
         """
-        package_to_check = 'cloud-init'
+        cloud_cfg = '/etc/cloud/cloud.cfg'
+        verify_cmd = f'rpm -Vf {cloud_cfg} | grep -e "^S.5.*{cloud_cfg}$"'
```

Comment:
I think that you should consider also files shipped by the package in /etc/cloud/cloud.cfg.d`

Reply:
Sounds like writing completely different test
Of course, no problem for me

Reply:
Fixed in 16aedb1cc4427de8879b287ed0db459c9af76639

Reply:
> Sounds like writing completely different test Of course, no problem for me

True, but the test originally covered also `/etc/cloud/cloud.cfg.d`, even though it was probably not the intention based on the doc string.